synfig-studio/src/gui/app.cpp

```diff
@@ -3826,14 +3827,17 @@ App::dialog_paragraph(const std::string &title, const std::string &message,std::
 
 	Gtk::Label* label = manage(new Gtk::Label(message));
 	label->show();
-	dialog.get_content_area()->pack_start(*label);
+	dialog.get_content_area()->pack_start(*label, false, false);
 
 	Glib::RefPtr<Gtk::TextBuffer> text_buffer(Gtk::TextBuffer::create());
 	text_buffer->set_text(text);
 	Gtk::TextView text_view(text_buffer);
 	text_view.show();
+	Gtk::ScrolledWindow s_window;
+	s_window.add(text_view);
+	s_window.show();
 
-	dialog.get_content_area()->pack_start(text_view);
+	dialog.get_content_area()->pack_start(s_window);
```

Comment:
I suggest to increase default window size:

Current:
![Screenshot_98](https://user-images.githubusercontent.com/5604544/187342906-269d7dfc-c95b-4dfc-b040-7f722b6056c5.png)

Suggested:
![Screenshot_97](https://user-images.githubusercontent.com/5604544/187342919-3bc4b2af-5bfa-4741-9fd0-4440c995c99c.png)

Reply:
done